packages/client/README.md

```diff
@@ -14,26 +14,18 @@ import { Web3Storage } from 'web3.storage'
 // Construct with token and endpoint
 const client = new Web3Storage({ token: apiKey })
 
-// Pack files into a CAR
-client.pack(files: Iterable<File>): Promise<{car: Blob, root: CIDString}>
-
-// Store a CAR
-client.store(car: Blob): Promise<CIDString>
-
-// Send the files, packing them into a CAR under the hood
-client.packAndStore(files: Iterable<File>): Promise<CIDString>
+// Pack files into a CAR and send to web3.storage
+const rootCid = await client.put(fileInput.files) // Promise<CIDString>
 
 // Get info on the Filecoin deals that the CID is stored in
-client.status(cid: string): Promise<Metadata>
-
-// Fetch a CAR over http
-client.get(cid: string): Promise<Blob | null>
-
-// Unpack and verify a CAR
-client.unpack(car: Blob): AsyncIterable<?>
-
-// Fetch, verify and unpack the CID to files
-client.getAndUnpack(cid: string): AsyncIterable<?>
+const info = await client.status(rootCid) // :Promise<Metadata>
+
+// Fetch and verify files from web3.storage
+const res = await client.get(rootCid) // :Promise<CarResponse>
```

Comment:
ups, rpobably this should be Web3Response?

Reply:
It is done!